roles/mongodb-server/tasks/auth_initialization.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: backup mongod.conf
```

Comment:
yes. for new env installation, the admin user should be added before enable auth and replicaset.